README.md

```diff
@@ -1,26 +1,24 @@
 # Node.js API (N-API)
-This repository is the home for ABI Stable Node API project (N-API). The goal of this
-project is to provide a stable Node API for native module developers. N-API aims
-to provide ABI compatibility guarantees across different Node versions and also
-across different Node VMs – allowing N-API enabled native modules to just work
+This repository is the home for ABI Stable Node API project (N-API).
+The goal of this project is to provide a stable Node API for native
+module developers. N-API aims to provide ABI compatibility guarantees
+across different Node versions and also across different Node
+VMs - allowing N-API enabled native modules to just work
 across different versions and flavors of Node.js without recompilations.
 
 It is introduced by this Node enhancemnet proposal:
 [005-ABI-Stable-Module-API.md](https://github.com/nodejs/node-eps/blob/master/005-ABI-Stable-Module-API.md).
 
-This is project is in experimental stage at the moment. During the latest [VM Summit](https://github.com/nodejs/vm/issues/4), we [discussed the progress](https://github.com/nodejs/abi-stable-node/blob/doc/VM%20Summit.pdf) of this project. The summit participants agreed to consider a PR for this feature as experimental in the upcoming Node.js version 8.0. You can checkout the [Notes from Node.js VM Summit](https://blogs.windows.com/msedgedev/2017/03/13/notes-from-nodejs-vm-summit/) for more details.
+N-API is now part of core as an experimental feature. Documentation is available here:
+[https://nodejs.org/docs/latest/api/n-api.html]().
 
 **Branches**
```

Comment:
Also, we should update the "How to get involved section" from the text in the blog.